app/models/application.rb

```diff
@@ -0,0 +1,14 @@
+class Application < ApplicationRecord
+  has_many :pet_applications
+  has_many :pets, through: :pet_applications
+  validates :name, :street_address, :city, :state, :zip_code, :description, presence: true
+  #enum application_status: {"In Progress": 0, "Pending": 1, "Accepted":2, "Rejected":3}
+  
+  def in_progress?
+    application_status == "In Progress"
+    #use an enum vs a method at the same level as the has_many
```

Comment:
This is a great option when your field is a string datatype! If your field is an enum type, you get this method created out of the box. For example, with an enum, every enum value gets turned automatically into a boolean method that will return true if the field contains that value. So you would be able to call `application.accepted?` or `application.pending?` without having to implement those methods because they are made for you when using an enum